octobot_trading/exchanges/connectors/ccxt_exchange.py

```diff
@@ -301,91 +493,474 @@ async def get_all_currencies_price_ticker(self, **kwargs: dict) -> typing.Option
+            return order
+        if enums.CCXTExchangeConfigMethods.GET_ORDER_PRIVATE.value in defined_methods and \
+                (order := await self.get_order_private(order_id, symbol,
+                                                       check_completeness=check_completeness, **kwargs)):
+            return order
+        if enums.CCXTExchangeConfigMethods.GET_ORDER_FROM_OPEN_AND_CLOSED_ORDERS.value in defined_methods and \
+                (order := await self.get_order_from_open_and_closed_orders(order_id, symbol,
+                                                                           check_completeness=check_completeness,
+                                                                           **kwargs)):
+            return order
+        if enums.CCXTExchangeConfigMethods.GET_ORDER_USING_STOP_ID.value in defined_methods and \
+                (order := await self.get_order_using_stop_id(order_id, symbol, check_completeness=check_completeness)):
+            return order
+        if enums.CCXTExchangeConfigMethods.GET_ORDER_FROM_TRADES.value in defined_methods and \
+                (order := await self.get_order_from_trades(order_id, symbol, check_completeness=check_completeness)):
+            return order
+        self.logger.debug(f"Order not found using get_order: {order_id} / {symbol} - order might not exist anymore")
+
+    async def get_order_default(self, order_id: str, symbol: str = None, check_completeness: bool = True,
+                                **kwargs: dict) -> typing.Optional[dict]:
+        if self.client.has.get('fetchOrder'):
             try:
                 with self.error_describer():
                     params = kwargs.pop("params", {})
-                    return await self.client.fetch_order(order_id, symbol, params=params, **kwargs)
-                # self.exchange_manager.exchange_personal_data.upsert_order(order_id, updated_order) TODO
+                    return await self.parse_order(
+                        await self.client.fetch_order(order_id, symbol, params=params, **kwargs),
+                        check_completeness=check_completeness, )
             except ccxt.OrderNotFound:
                 # some exchanges are throwing this error when an order is cancelled (ex: coinbase pro)
                 pass
             except ccxt.NotSupported as e:
-                # some exchanges are throwing this error when an order is cancelled (ex: coinbase pro)
-                raise octobot_trading.errors.NotSupported from e
-        else:
-            # When fetch_order is not supported, uses get_open_orders and extract order id
-            open_orders = await self.get_open_orders(symbol=symbol)
+                self.logger.error(f"Failed to fetch closed orders using get_order_default: Not Supported {e}")
+            except Exception as e:
+                self.logger.error(f"Failed to fetch closed orders using get_order_default: {e}")
```

Comment:
```suggestion
                self.logger.exception(e, True, f"Unexpected error when orders using get_order_default: {e}")
```

Reply:
when I do logger.exception, it is not raising an error I guess?